### ISSUE TITLE
In offline mode replace inline plotlyjs with import

### ISSUE THREAD

**Issue by sverhoeven** [haddocking/haddock3#1038]
## **Desired feature/enhancement**

Running a caprieval analysis in offline mode uses lots of diskspace.

For example I ran https://github.com/haddocking/haddock3/blob/main/examples/scoring/capri-scoring-test.cfg with `offline = true`.
<details>
<summary>The capriscoring-test/analysis/7_caprieval_analysis directory uses 136Mb.</summary>

```
3.6M    air_clt.html
3.6M    bsa_clt.html
4.0K    capri_clt.tsv
4.0K    capri_ss.tsv
3.6M    desolv_clt.html
3.6M    dockq_air.html
3.6M    dockq_clt.html
3.6M    dockq_desolv.html
3.6M    dockq_elec.html
3.6M    dockq_score.html
3.6M    dockq_vdw.html
3.6M    elec_clt.html
3.6M    fnat_air.html
3.6M    fnat_clt.html
3.6M    fnat_desolv.html
3.6M    fnat_elec.html
3.6M    fnat_score.html
3.6M    fnat_vdw.html
3.6M    ilrmsd_air.html
3.6M    ilrmsd_clt.html
3.6M    ilrmsd_desolv.html
3.6M    ilrmsd_elec.html
3.6M    ilrmsd_score.html
3.6M    ilrmsd_vdw.html
3.6M    irmsd_air.html
3.6M    irmsd_clt.html
3.6M    irmsd_desolv.html
3.6M    irmsd_elec.html
3.6M    irmsd_score.html
3.6M    irmsd_vdw.html
3.6M    lrmsd_air.html
3.6M    lrmsd_clt.html
3.6M    lrmsd_desolv.html
3.6M    lrmsd_elec.html
3.6M    lrmsd_score.html
3.6M    lrmsd_vdw.html
7.2M    report.html
3.6M    score_clt.html
160K    summary.tgz
3.6M    vdw_clt.html
```

</details>

It would be nice if offline caprieval analysis html pages would use less space.

In #1025 I am importing report.bundle.js file so each caprieval module can reuse that js file. I think the same idea can be applied to the plotly at https://github.com/haddocking/haddock3/blob/64d625bce6826be0164bc678544cd420bc9f49ef/src/haddock/libs/libplots.py#L131
By instead of embedding the plotlyjs write it to a file once and importing it. 

## **Motivation**

The upside is going from 136Mb to 3.6Mb (the size of a single plotly js file)
The downside is that you need a web server to view all the html files.

**Comment by VGPReys** [comment]
Thanks for catching this issue @sverhoeven 

The #1041 PR quite reduces the size of it by quite a lot!

<details>
<summary>The capriscoring-test/analysis/7_caprieval_analysis directory is now only 4.2Mb!.</summary>
```bash
891B capri_clt.tsv
3.5M plotly_bundle.js
8.7K irmsd_score.html
8.7K irmsd_desolv.html
8.7K irmsd_vdw.html
8.7K irmsd_elec.html
8.7K irmsd_air.html
8.7K dockq_score.html
8.7K dockq_desolv.html
8.7K dockq_vdw.html
8.7K dockq_elec.html
8.7K dockq_air.html
8.7K lrmsd_score.html
8.7K lrmsd_desolv.html
8.7K lrmsd_vdw.html
8.7K lrmsd_elec.html
8.7K lrmsd_air.html
8.7K ilrmsd_score.html
8.7K ilrmsd_desolv.html
8.7K ilrmsd_vdw.html
8.7K ilrmsd_elec.html
8.7K ilrmsd_air.html
8.7K fnat_score.html
8.7K fnat_desolv.html
8.7K fnat_vdw.html
8.7K fnat_elec.html
8.7K fnat_air.html
8.3K score_clt.html
8.3K vdw_clt.html
8.3K elec_clt.html
8.3K air_clt.html
8.3K desolv_clt.html
8.3K irmsd_clt.html
8.3K lrmsd_clt.html
8.3K ilrmsd_clt.html
8.3K fnat_clt.html
8.3K dockq_clt.html
8.4K bsa_clt.html
56K report.html
160K summary.tgz
707B capri_ss.tsv
```

**Comment by VGPReys** [comment]
Closed by #1038
